CONTRIBUTING.md

```diff
@@ -39,24 +39,20 @@ most effective at receiving input and making progress.
 If the feature is **small** - a change to a single piece of functionality, or an
 addition that can be expressed clearly and succinctly in a few sentences, then
 the most appropriate place to propose it is as a [new Feature request] in this
-repository. Such issues will typically receive `p:3` priority in their initial
-triage.
+repository.
 
 If the feature is **more complicated**, involves protocol changes, or has
-potential safety or performance implications, then first check with us whether
-it is something we are willing to consider and is in-scope for our project.
+potential safety or performance implications, then consider [proposing an ADR]
+and submit it as a pull request ot this repository. This will allow a structured
+review and commenting of the proposed changes. You should aim to reach consensus
```

Comment:
Consensus among who? Any committers or contributors involved in the discussion?

Reply:
This was there before, I guess consensus in general, the actual decision making will be made more clear in the governance document (which is not there yet). I'll do some rewording to make this more clear.

Reply:
Replaced with:
> You should aim to get the ADR accepted and merged before starting on implementation.